Cargo.toml

```diff
@@ -105,9 +105,10 @@ pyproject-toml = { version = "0.9.0" }
 quick-junit = { version = "0.4.0" }
 quote = { version = "1.0.23" }
 rand = { version = "0.8.5" }
+rustc-hash = { version = "2.0.0" }
```

Comment:
I left it in red-knot, it was giving me some trouble around lack of a `Default` impl.

Reply:
We should resolve this before merging